packages/contracts/src/handlers/UnclaimsHandler.sol

```diff
@@ -442,14 +468,14 @@ contract UnclaimsHandler is ReentrancyGuard, Ownable {
         require(success, "ETH transfer to us.sender failed");
         payable(msg.sender).transfer(consumedGas * maxFeePerGas);
 
-        emit EmailWalletEvents.UnclaimedStateVoided(emailAddrCommit, us.sender);
+        emit EmailWalletEvents.UnclaimedStateVoided(id, us.emailAddrCommit, us.sender);
     }
 
-    function getSenderOfUnclaimedFund(bytes32 emailAddrCommit) public view returns (address) {
-        return unclaimedFundOfEmailAddrCommit[emailAddrCommit].sender;
+    function getSenderOfUnclaimedFund(uint256 id) public view returns (address) {
```

Comment:
Are these methods still needed? They were used to check if UF/US exists in core contract (and nowhere else I think), which you have removed now.

(Btw this could have been `isUnclaimedFundExists` and `isUnclaimedStateExists` and return bool)

Reply:
Thank you for your review!
I changed those methods to getSenderOfUnclaimedFund/getSenderOfUnclaimedState which returns the struct directly because the public getter functions don't return the struct directly.
https://github.com/zkemail/email-wallet/pull/21/commits/0bae917aa9f8da91566af59cc3b168072f60eff7